python/ml4ir/base/config/parse_args.py

```diff
@@ -401,6 +401,16 @@ def define_args(self):
             help="The number of folds for K-fold Cross Validation. Must be > 2 if testset is included in folds and > 1 otherwise.",
         )
 
+        self.add_argument(
+            "--monte_carlo_inference_trials",
+            type=int,
+            default=0,
+            help="""Adding stochastisity at inference time. Monte Carlo trials is the number of times inference will be run while using dropout during inference.
+            The final output scores will be the aggregate sum of scores produced for each monte carlo trial (run).
+            This argument should be used only if there is a non zero dropout probability"""
+
+        )
```

Comment:
@mohazahran might be better to just roll this into the model config, no?

Reply:
will also help with the deterministic follow up.

Reply:
I think having it either way is fine.